tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/About.java

```diff
@@ -99,7 +99,14 @@ public Response getAbout() {
                 r.put("dbInfo", d.getDBInfo());
             }
         }
-
+        r.put("memory", getMemoryStats());
         return Response.ok(r).build();
     }
+
+    private static String getMemoryStats() {
+        Runtime rt = Runtime.getRuntime();
+        return "total: " + rt.totalMemory() +
+            "; max: " + rt.maxMemory() +
+            "; free: " + rt.freeMemory();
+    }
 }
```

Comment:
instead, please send individual numbers instead of a string, so that we can programmatically track it!  Then, assemble the string on the client side.

Reply:
also #893 (2 years ago) removed these fields from the `SurveyAjax?what=status` endpoint -> https://github.com/unicode-org/cldr/pull/893/files#diff-d994bfe2804309e4400822a8cd2bed66b3a1e05ab5ef0f518a81e9738af6ee70L1317

Can you reinstate those?

Reply:
I'll open a separate PR

Reply:
Opened https://github.com/unicode-org/cldr/pull/1845

Reply:
I'll make a subsequent PR to split up the memory stats